modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -4300,13 +4303,13 @@ def map_fn(df):  # pragma: no cover
         # than it would be to reuse the code for specific columns.
         if len(columns) == len(self.columns):
             new_modin_frame = self._modin_frame.apply_full_axis(
-                0, map_fn, new_index=self.index
+                0, map_fn, new_index=self.index, dtypes=bool
```

Comment:
an example of how we can use 'remaining_dtype' functionality, later we should find more places where we can apply this